services/telegram/service.go

```diff
@@ -0,0 +1,81 @@
+package telegram
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Service struct {
+	chat_id          string
+	url              string
+	global           bool
+	stateChangesOnly bool
+	logger           *log.Logger
+}
+
+func NewService(c Config, l *log.Logger) *Service {
+	return &Service{
+		chat_id:          c.ChatId,
+		url:              c.URL + c.Token + "/sendMessage",
+		global:           c.Global,
+		stateChangesOnly: c.StateChangesOnly,
+		logger:           l,
+	}
+}
+
+func (s *Service) Open() error {
+	return nil
+}
+
+func (s *Service) Close() error {
+	return nil
+}
+
+func (s *Service) Global() bool {
+	return s.global
+}
+func (s *Service) StateChangesOnly() bool {
+	return s.stateChangesOnly
+}
+
+func (s *Service) Alert(chatId, message string) error {
+	if chatId == "" {
+		chatId = s.chat_id
+	}
+	postData := make(map[string]interface{})
+	postData["chat_id"] = chatId
+	postData["text"] = message
```

Comment:
They have [no server](https://telegram.org/faq#q-can-i-run-telegram-using-my-own-server) available in open source, only clients. So I'm afraid we can only can mock it in tests.